node/cn/tracers/api.go

```diff
@@ -743,7 +715,7 @@ func (api *CommonAPI) standardTraceBlockToFile(ctx context.Context, block *types
 	if config != nil && config.Reexec != nil {
 		reexec = *config.Reexec
 	}
-	statedb, release, err := api.backend.StateAtBlock(ctx, parent, reexec, nil, true, false)
+	_, _, _, statedb, release, err := api.backend.StateAtTransaction(ctx, parent, 0, reexec)
```

Comment:
Maybe we should do the same for the TraceCall.
1. For correctness: TraceCall should simulate a transaction execution after Initialize().
2. For future extensibility: https://github.com/ethereum/go-ethereum/blob/da17f2d65b8922a20f86740644b2d7f6ef688e9b/eth/tracers/api.go#L948-L952 geth eth_traceCall RPC takes optional TxIndex argument. If we're going to adopt the feature, it has to be implemented with StateAtTransaction.

Reply:
1. I'll replace `StateAtBlock` to `StateAtTransaction` with tx index zero.
2. This seems to be an independent feature in this PR. Do you want me to add that feature together within this PR?

Reply:
Sorry I didn't mean to add the feature now. It might be nice to replace all call sites of StateAtBlock with StateAtTransaction so later we can be more comfortable with introducing the feature.

Reply:
PTAL https://github.com/kaiachain/kaia/pull/121/commits/a9f4650f134984439abfbbd8650a9c44d8616e70